.bazelrc

```diff
@@ -84,46 +126,103 @@ build:windows --linkopt=/OPT:REF
 build:windows --host_linkopt=/OPT:REF
 build:windows --linkopt=/OPT:ICF
 build:windows --host_linkopt=/OPT:ICF
-build:windows --copt=/d2ReducedOptimizeHugeFunctions
-build:windows --host_copt=/d2ReducedOptimizeHugeFunctions
-build:windows --experimental_strict_action_env=true
 
 # Verbose failure logs when something goes wrong
 build:windows --verbose_failures
 
-# Suppress C++ compiler warnings, otherwise build logs become 10s of MBs.
-build:windows --copt=/W0
-
 # On windows, we never cross compile
 build:windows --distinct_host_configuration=false
 
-build --define=use_fast_cpp_protos=true
-build --define=allow_oversize_protos=true
+# Options to build TensorFlow 1.x or 2.x.
+build:v1 --define=tf_api_version=1 --action_env=TF2_BEHAVIOR=0
+build:v2 --define=tf_api_version=2 --action_env=TF2_BEHAVIOR=1
 
-# Enable using platform specific build settings, except when cross-compiling for
-# mobile platforms.
-build --enable_platform_specific_config
-build:android --noenable_platform_specific_config
+# Flag to enable remote config
+common --experimental_repo_remote_exec
 
-# Android configs. Bazel needs to have --cpu and --fat_apk_cpu both set to the
-# target CPU to build transient dependencies correctly. See
-# https://docs.bazel.build/versions/master/user-manual.html#flag--fat_apk_cpu
-build:android --crosstool_top=//external:android/crosstool
-build:android --host_crosstool_top=@bazel_tools//tools/cpp:toolchain
-build:android_arm --config=android
-build:android_arm --cpu=armeabi-v7a
-build:android_arm --fat_apk_cpu=armeabi-v7a
-build:android_arm64 --config=android
-build:android_arm64 --cpu=arm64-v8a
-build:android_arm64 --fat_apk_cpu=arm64-v8a
-build:android_x86 --config=android
-build:android_x86 --cpu=x86
-build:android_x86 --fat_apk_cpu=x86
-build:android_x86_64 --config=android
-build:android_x86_64 --cpu=x86_64
-build:android_x86_64 --fat_apk_cpu=x86_64
+# Address sanitizer
```

Comment:
Very nice that this is now included by TensorFlow itself (right?), I'm a big fan of the [Address Sanitizer](https://clang.llvm.org/docs/AddressSanitizer.html). We could later consider running some of our CI tests with the `asan` target such that they benefit from all kinds of checks (at a ~2x runtime overhead).

Reply:
Yeah I was pleasantly surprised to see this too, and I agree we should definitely at some point set up some CI using it :)